pkg/asset/installconfig/aws/validation.go

```diff
@@ -313,10 +323,24 @@ func validateServiceEndpoints(fldPath *field.Path, region string, services []aws
 	if err := utilerrors.NewAggregate(errs); err != nil {
 		allErrs = append(allErrs, field.Invalid(fldPath, services, err.Error()))
 	}
-
 	return allErrs
 }
 
+// validateRegion checks if the given region is supported by AWS.
+func validateRegion(region string) error {
+	ses, err := GetSession()
```

Comment:
We need a way to sub out the as calls for the unit tests, right?

Reply:
Not sure. The aws calls are working in CI.

Reply:
I think that what you are seeing is the call to `GetSession` throwing an error that the calling code is interpreting as an unknown region.